### DISCUSSION TITLE
How to get stable-diffusion-xl-base-1.0 to run

### DISCUSSION THREAD

**Discussion by manuelkamp** [mudler/LocalAI#1229] [Q&A]
I want to be able to generate 1024x1024 images and figured out, with the stablediffusion from gallery it supports only 512x512. I found now stable-diffusion-xl-base-1.0 on huggingface, but I have no clue how to get it running in LocalAI nor if it even supports gguf?

**Comment by manuelkamp** [comment]
noone ever tried to use something else for image generation here?

**Reply by kopierschnitte** [reply]
Well I tried the stabilityai model but it took about 30min (!) to generate an image, even after download was completed.
Automatic1111 does the same within 30sec.

The sd-model in the mentioned HowTo works a lot faster

**Comment by lunamidori5** [comment]
https://localai.io/howtos/easy-setup-sd/ <3

**Reply by NW15D** [reply]
404 err

**Reply by lunamidori5** [reply]
@neowisard the link got changed its now found here - https://io.midori-ai.xyz/howtos/easy-setup-sd/

**Comment by mudler** [comment]
The setup in the docs ( https://localai.io/model-compatibility/diffusers/#model-setup ) shows SDXL, that should output images bigger than 512x512 . Did you tried the example in the docs?

**Reply by NW15D** [reply]
![Screenshot_20240123_125149_Chrome](https://github.com/mudler/LocalAI/assets/1011406/1e70c8b7-4397-4f16-99ac-d3e59b0e590c)

**Reply by lunamidori5** [reply]
https://localai.io/features/image-generation/
https://io.midori-ai.xyz/howtos/easy-setup-sd/
